tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -32,7 +32,7 @@ Feature: display image in media viewer on the webUI
     And the public views the file "testavatar.jpg" in the media viewer using the webUI
     Then the file "testavatar.jpg" should be displayed in the media viewer webUI
 
-  @skipOnOCIS
+#  @skipOnOCIS
```

Comment:
If this test passes we can remove this tag line completely.

Reply:
fixed